src/Storage/StorageInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\RateLimiter\Storage;
+
+interface StorageInterface
```

Comment:
It would be nice to have at least one implementation out of the box.

Reply:
We can move cache back to non-dev, I think.